packages/main/src/TableUtils.ts

```diff
@@ -17,9 +17,61 @@ const findRowInPath = (composedPath: Array<EventTarget>) => {
 	return composedPath.find((el: EventTarget) => el instanceof HTMLElement && el.hasAttribute("ui5-table-row")) as TableRow;
 };
 
+const findScrollContainer = (element: HTMLElement): HTMLElement => {
+	while (element) {
+		const { overflowY } = window.getComputedStyle(element);
+		if (overflowY === "auto" || overflowY === "scroll") {
+			return element;
+		}
+
+		if (element.parentNode instanceof ShadowRoot) {
+			element = element.parentNode.host as HTMLElement;
+		} else {
+			element = element.parentElement as HTMLElement;
+		}
+	}
+
+	return document.scrollingElement as HTMLElement || document.documentElement;
+};
+
+const scrollElementIntoView = (scrollContainer: HTMLElement, element: HTMLElement, stickyElements: HTMLElement[]) => {
+	if (stickyElements.length === 0) {
+		return;
+	}
+
+	const elementRect = element.getBoundingClientRect();
+
+	const { x: stickyX, y: stickyY } = stickyElements.reduce(({ x, y }, stickyElement) => {
+		const { top, left } = getComputedStyle(stickyElement);
+		const stickyElementRect = stickyElement.getBoundingClientRect();
+		if (top !== "auto" && stickyElementRect.bottom > elementRect.top) {
```

Comment:
how computedStyle can return auto?

Reply:
If no right/left is set, computedStyle returns "auto".